packages/block-editor/src/utils/dom.js

```diff
@@ -86,3 +86,24 @@ export function isInsideRootBlock( blockElement, element ) {
 export function hasInnerBlocksContext( element ) {
 	return !! element.querySelector( '.block-editor-block-list__layout' );
 }
+
+/**
+ * Finds the block client ID given any DOM node inside the block.
+ *
+ * @param {Node} node DOM node.
+ *
+ * @return {?string} Client ID or undefined if the node is not part of a block.
+ */
+export function getBlockClientId( node ) {
+	if ( node.nodeType !== node.ELEMENT_NODE ) {
```

Comment:
Unless I'm mistaken, this should be:

```suggestion
	if ( node.nodeType !== Node.ELEMENT_NODE ) {
```

https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#Constants

Currently, I believe it will always be triggering the condition, since `node.ELEMENT_NODE` would evaluate to `undefined`.

Reply:
I think this is correct. Every node instance has these properties.

Reply:
> I think this is correct. Every node instance has these properties.

Ah, interesting. I did not know this.